src/test/java/NEWTsPresenterTest.java

```diff
@@ -0,0 +1,66 @@
+import io.reactivex.Observable;
+import io.reactivex.plugins.RxJavaPlugins;
+import io.reactivex.schedulers.Schedulers;
+import json.Spell;
+import json.SpellList;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+class NEWTsPresenterTest
+{
+    @BeforeAll
+    public static void beforeAllTests()
+    {
+        // the following code makes tests run single-threaded
+        RxJavaPlugins.setIoSchedulerHandler((scheduler) -> Schedulers.trampoline());
+        RxJavaPlugins.setNewThreadSchedulerHandler((scheduler) -> Schedulers.trampoline());
+
+        // this will run one time before all tests in this class
+    }
+
+    @Test
+    void resetFlashCard()
+    {
+
+    }
+
+    @Test
+    void loadSpellInformation()
+    {
+        //given
+        NEWTsPracticeExam view = mock(NEWTsPracticeExam.class);
+        SpellGenerator model = mock(SpellGenerator.class);
+        NEWTsPresenter presenter = new NEWTsPresenter(view, model);
+        SpellList spellList = mock(SpellList.class);
+
+       doReturn("Conjures water").when(spellList.get(0)).getEffect();
```

Comment:
Your test is failing on this line.
Can you see why?

Reply:
Was it because I assumed spellList would have a zero'th element?

Reply:
How do I fix that? An if statement?

Reply:
Why doesn't `spellList` have a 0th element?

Reply:
because it's a mock?

Reply:
So how can you fix that?